refextract/references/kbs/report-numbers.kb

```diff
@@ -268,13 +268,20 @@ CMS PAS EWK            ---CMS-PAS-EWK
 CMS PAS BTV            ---CMS-PAS-BTV
 CMS PAS FWD            ---CMS-PAS-FWD
 CMS PAS TRK            ---CMS-PAS-TRK
-CMS PAS SMP            ---CMS-PAS-SMP
 CMS PAS PFT            ---CMS-PAS-PFT
 CMS PAS MUO            ---CMS-PAS-MUO
 CMS PAS JME            ---CMS-PAS-JME
 CMS PAS EGM            ---CMS-PAS-EGM
 CMS PAS DIF            ---CMS-PAS-DIF
 CMS PAS B2G            ---CMS-PAS-B2G
+CMS PAS LUM            ---CMS-PAS-LUM
+CMS PAS FTS            ---CMS-PAS-FTA
```

Comment:
is this correct, changing FTS to FTA ?

Reply:
I note that neither `CMS-PAS-FTA` nor `CMS-PAS-FTS` do appear to be used. See http://cdsweb.cern.ch/collection/CMS%20Physics%20Analysis%20Summaries?ln=en&as=1

search for those report number prefixes turns up empty